de/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,36 +33,36 @@
 # the terms of any one of the MPL, the GPL or the LGPL.
 #
 # ***** END LICENSE BLOCK *****
-IncorrectKey_TryAgain? = Master password incorrect. Do you want to try again?
-WantToCaptureForm? = Do you want to remember the values you filled in?
-WantToSavePasswordObscured? = Password Manager can remember this logon and enter it automatically the next time you return to this website.##Do you want Password Manager to remember this logon?
-WantToSavePasswordEncrypted? = Password Manager can remember this logon and enter it automatically the next time you return to this website.##Do you want Password Manager to remember this logon?
-xWantToSavePasswordObscured? = Do you want to remember your user name and password?##Stored values are not password protected.
-xWantToSavePasswordEncrypted? = Do you want to remember your user name and password?##Stored values are password protected.
+IncorrectKey_TryAgain? = Master Passwort inkorrect. Möchten Sie es erneut versuchen?
+WantToCaptureForm? = Möchten Sie die eingegebenen Werte speichern?
+WantToSavePasswordObscured? = Password Manager kann diese Anmeldung speichern und die Anmeldedaten beim nächsten Mal automatisch eingeben, wenn Sie zu dieser Webseite zurückkehren.##Do you want Password Manager to remember this logon?
```

Comment:
CORRECTIONS NEEDED +WantToSavePasswordObscured? = Password-Manager kann diese Anmeldung speichern und die Anmeldedaten beim nächsten Mal automatisch eingeben, wenn Sie zu dieser Webseite zurückkehren.##Do you want Password Manager to remember this logon?